### DISCUSSION TITLE
Tutorial 3: Multithreaded HPL Runs Using Your Custom Compiled OpenBLAS and OpenMPI Libraries

### DISCUSSION THREAD

**Discussion by nyameko** [chpc-tech-eval/scc#61] [Show and tell]
Post your `htop` or `btop` screenshots of your multithreaded HPL runs.

![MultiThreaded_HPL](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/21206557/2a751ac3-7356-49f3-ae1b-d3d3ad27e55a)

**Comment by Talhahpatelia** [comment]
## Wits-A

Image of multithreaded HPL run:

<img width="696" alt="Screenshot 2024-07-10 153528" src="https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/29210701/7ef49826-1b54-4b88-852f-48c2db39539a">

**Comment by CraftierLamb599** [comment]
# Wits-1
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/137203651/1fbb0fef-516e-467f-946b-c0586ad06560)

**Comment by Ghost-3342** [comment]
### HPC Wizards
![ALL_CPUS-100](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/61750126/a0f116c7-a7a9-47ec-b3d5-d03cc66b1031)

**Reply by Ghost-3342** [reply]
Had to edit the Makefile

**Reply by n-zwane** [reply]
Technocrats (UWC)
<img width="1920" height="1080" alt="tmux-benchmark" src="https://github.com/user-attachments/assets/67b246c6-a7ad-4fef-9984-f9903f983afc" />

**Comment by ADELAIDEMOGALE** [comment]
### **Khulisa Data**

![Screenshot (700)](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/166052302/20eef1d3-270f-4bc0-8eb9-d78b4f847548)

**Comment by MishkaSingh1** [comment]
**EEElite(UCT)**

![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175049735/07aeaff9-f903-4ca2-827f-de3ab9890cd7)

**Comment by Kohzmik** [comment]
CUT_Team
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/138365090/fd909363-dbb0-495c-9743-2b9acc95e5d1)

**Comment by SfisoMncube** [comment]
khulisa IP-A
![image_png](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/151764841/b180e242-37c9-4551-8bad-bd74f68ad312)

**Comment by prince-taker** [comment]
Ctrl Alt Elite
![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/f0d83084-a07e-43df-94cf-735ca006c429)

**Comment by Dc-Spears** [comment]
**CPUT! (DC SPEARS)**
![CPUT-HPL-all-CPUs](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/175037009/0c0c8589-fb7e-47ea-9a05-eae510ddd7fd)

**Comment by Rosemary2301** [comment]
Algorithm Avengers
![htop](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/2eacc163-05f1-47c2-9af3-37c3fb9c9c31)
![btop](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/143855810/0745fed0-991b-4330-a61c-b05a83a1dab0)

**Comment by WilliamMohlahlane** [comment]
### **ThunderBytes**

![WhatsApp Image 2024-07-12 at 02 09 39](https://github.com/user-attachments/assets/d881c8f6-0172-40c2-bfe5-f87aa075b1cd)

**Comment by nexusWorkers** [comment]
![Screenshot 2024-07-12 025234](https://github.com/user-attachments/assets/2409cb7c-7810-41f3-af4c-54fe9c6cab83)

**Comment by hoperiley0806** [comment]
**Parallel **Pioneers****

![Screenshot (134)](https://github.com/user-attachments/assets/6c66d13b-8aa9-4b12-8ed5-55d1befcfc79)

**Comment by SizweMarole-21389** [comment]
![btop and top](https://github.com/user-attachments/assets/2a88e1d4-c089-4705-8cd5-00ee64621653)
CYBERSHIELD

**Comment by ruvarashecodes** [comment]
CtrlAltDefeat
![100%](https://github.com/user-attachments/assets/e13458f7-bcb9-45e3-a327-89197121d734)
#hardatwork
#CtrlAltDefeat

**Comment by DCspearsv2** [comment]
DC Spears
![Screenshot 2025-07-09 165108](https://github.com/user-attachments/assets/03ef9c98-9a48-4ece-9680-416d401a5409)

**Comment by KamvaMj** [comment]
DC Daggers

![Screenshot (77)](https://github.com/user-attachments/assets/40ceba63-2431-4041-82c6-ecc2b10c519b)

**Comment by SFBdragon** [comment]
0v1rcl0ck1d

![image](https://github.com/user-attachments/assets/71d23342-62f8-4f24-997f-1331fbd3080a)

**Comment by Mwelase-bit** [comment]
DebugThugs
![Btop OpenBlas22](https://github.com/user-attachments/assets/f8e4412e-3c40-456a-939e-651e2c19758c)

**Comment by UNISA-SCT25** [comment]
![Tut3](https://github.com/user-attachments/assets/a9730659-42b9-4e43-b212-b7d827cf4654)

**Comment by Moagi99** [comment]
TUT_X

![Screenshot (20)](https://github.com/user-attachments/assets/1fa7029b-d28a-4245-8bc1-f60916c0d970)

**Comment by ShavirV** [comment]
TuksByters
<img width="1919" height="1079" alt="image" src="https://github.com/user-attachments/assets/b3db965f-0451-4901-9bf3-52808f9a576f" />

**Comment by MphephuSamuel** [comment]
computer champions

<img width="1920" height="1080" alt="Screenshot (995)" src="https://github.com/user-attachments/assets/f07a3b6f-c57d-4180-bbc3-8d8982b17eb5" />

**Comment by Cloud-Strive** [comment]
Runtime Blinders

<img width="1920" height="1080" alt="Screenshot (16)" src="https://github.com/user-attachments/assets/14616299-84ec-4631-a01e-0c3af75329c3" />
<img width="1920" height="1080" alt="Screenshot (17)" src="https://github.com/user-attachments/assets/b79c7fe3-ac59-453e-9fc6-4ec2d99e9496" />

**Comment by Kevin-Ebrahim** [comment]
![bench_tut_3_SOURCED](https://github.com/user-attachments/assets/17562bba-d1e7-40e6-9e2e-fb0f235d418e)
Wits-A

**Comment by ethannef** [comment]
Wits1
![tut3 image 1 xpl](https://github.com/user-attachments/assets/d6b7d339-c942-4abc-88d4-0e6e7bba128d)

**Comment by CadmusKei** [comment]
Team Jargon (UWC) | SCC 2026 | Multithreaded HPL run using Custom Compiled OpenBLAS and OpenMPI Libraries
Edit: Better FLOPS

<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/f1c761ac-290d-4fff-8047-130bcc1d2e6f" />

**Comment by RoarkeBailey** [comment]
<img width="1856" height="1049" alt="Screenshot from 2026-07-07 17-37-02" src="https://github.com/user-attachments/assets/9446aa4d-d8b0-4d64-8cdb-e15b6cad4c73" />

G3MIKNIGHTS Multithreading HPL using OpenBLAS and OpenMPI libraries
calculated theoretical RPeak as 614.4 GFLOPS

**Comment by ShavirV** [comment]
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/15299a5f-a024-4892-b961-48607af82792" />

UPtime

**Comment by ThapeloVundla** [comment]
DC Spears

<img width="1920" height="1075" alt="Screenshot from 2026-07-08 12-39-30" src="https://github.com/user-attachments/assets/1bb15d9e-ca7d-42a3-b039-b5aca52b0482" />

**Comment by Leo-OB** [comment]
## DC Axes (CPUT) | SCC 2026
### Output for  multithreaded HPL run using custom compiled OpenBLAS and OpenMPI libraries
<img width="1854" height="1042" alt="HPL" src="https://github.com/user-attachments/assets/48bc70b2-d978-474f-ade3-f2de249c89f7" />

**Reply by Leo-OB** [reply]
### DC Axes (CPUT) | SCC 2026 Uploaded HPL highest benchmark:

<img width="1854" height="1042" alt="Highest_HPL_Benchmark" src="https://github.com/user-attachments/assets/9398ddc0-7133-400a-9a0d-ecf2331944c6" />

**Comment by JayJay-lab** [comment]
Sol_Plaatje_A
<img width="1920" height="1200" alt="image" src="https://github.com/user-attachments/assets/2d7c2425-d3f0-4a6b-bf32-178a82cf29f9" />

The theoritical peak was 611.84

**Comment by s225875411** [comment]
MadibazBits

<img width="1852" height="1018" alt="image" src="https://github.com/user-attachments/assets/a79f464f-0f37-410c-b1cf-a6ccdec49c9a" />

<img width="1852" height="1018" alt="image (1)" src="https://github.com/user-attachments/assets/b9af0aa5-a023-477f-9655-0df84878c30c" />

**Comment by ntando-memela** [comment]
WITS1-2026
<img width="1854" height="1048" alt="showHpl" src="https://github.com/user-attachments/assets/469ae27b-34b9-4976-a6e3-73a4f5963234" />

**Comment by Slindokuhle19** [comment]
UnizuluComputeX
<img width="1814" height="908" alt="Screenshot from 2026-07-08 21-46-37" src="https://github.com/user-attachments/assets/00e8b809-fe52-4f44-9da0-e136baa2412c" />

**Comment by BleSSednav3** [comment]
Resilient Geeks (uL)
<img width="1838" height="933" alt="Screenshot from 2026-07-08 22-36-06" src="https://github.com/user-attachments/assets/3002a643-68ce-4eef-b2c8-b9a2a910520c" />

**Comment by saxs-14** [comment]
3ncrypt3d (UMP) Btop multithreaded HPL run

<img width="1918" height="1078" alt="image" src="https://github.com/user-attachments/assets/5f4912b5-e860-4b4b-b641-71f6cfc8b19c" />
<img width="1918" height="1077" alt="Screenshot 2026-07-08 234114" src="https://github.com/user-attachments/assets/d0bd86b6-ab27-444f-a3b6-02cc460b60eb" />

**Comment by switchsquad** [comment]
<img width="1882" height="1001" alt="image" src="https://github.com/user-attachments/assets/652a3274-cbd2-40a3-ae63-067e4cd2daf8" />

Switch Squad (TUT)

**Comment by majaha8** [comment]
<img width="973" height="1015" alt="Screenshot from 2026-07-09 14-29-54" src="https://github.com/user-attachments/assets/328a5195-3652-421b-931b-6ccd323d32c8" />

**Comment by LufunoMiles** [comment]
Digital Dynamos [Htop or Btop]  <img width="908" height="942" alt="image" src="https://github.com/user-attachments/assets/190049d9-51ee-4fc2-88ce-2be50829fc9e" />
<img width="1834" height="942" alt="btop" src="https://github.com/user-attachments/assets/fceb0a49-1b5e-46f8-b829-d147d10a5922" />

**Comment by mabotenick97-cloud** [comment]
CyberOps
<img width="1600" height="846" alt="WhatsApp Image 2026-07-09 at 4 29 25 PM" src="https://github.com/user-attachments/assets/6ba7ba91-8878-4dc2-9429-d4d646f77212" />

**Comment by Lihle0512** [comment]
Chaos Compilers

<img width="1851" height="966" alt="Screenshot from 2026-07-10 09-08-55" src="https://github.com/user-attachments/assets/23a3b05b-6d8d-48fc-9ca7-305b092b9cd5" />
<img width="1851" height="966" alt="Screenshot from 2026-07-10 09-23-35" src="https://github.com/user-attachments/assets/aa55823e-ddb7-4ded-b818-11bfa46f4b33" />

**Comment by majaha8** [comment]
TEAM NEBULA 

<img width="517" height="545" alt="Screenshot_2026-07-09-18-09-53-25_6012fa4d4ddec268fc5c7112cbb265e7" src="https://github.com/user-attachments/assets/7eae725d-3538-40c0-b029-6d331ad07e90" />

**Comment by majaha8** [comment]
TAM NEBULA |CC 2026

<img width="947" height="1015" alt="hpl" src="https://github.com/user-attachments/assets/30286b26-31b6-49fd-9362-7690ddde5ed3" />

**Comment by bytedebuggers-dotcom** [comment]
**Byte Debuggers | UFH**
(btop)
<img width="1908" height="954" alt="Screenshot from 2026-07-09 20-59-18" src="https://github.com/user-attachments/assets/a4e98816-a77d-408f-b384-e02783b88338" />

<img width="813" height="454" alt="Screenshot from 2026-07-09 21-56-27" src="https://github.com/user-attachments/assets/8747ef01-20b6-4012-ab99-71c636d7b4c6" />

**Comment by majaha8** [comment]
Team nebula

<img width="404" height="270" alt="image" src="https://github.com/user-attachments/assets/0584edc9-ba20-4141-952b-61d5560068a2" />

**Comment by MrLampies** [comment]
DC Spears

<img width="1018" height="1035" alt="Screenshot from 2026-07-08 18-00-49" src="https://github.com/user-attachments/assets/a651beee-4e46-47ef-9f96-1d6069d63870" />

**Comment by majaha8** [comment]
TEAM NEBULA |2026

<img width="1853" height="1054" alt="good test" src="https://github.com/user-attachments/assets/1fd1085d-c15a-4ba9-8f87-bf7ec8d96e2c" />
